rebar.config.script

```diff
@@ -92,6 +92,9 @@ case VerList of
     _ -> ok
 end.
 
+[OptMajorVersion|_] = VerList.
+OtpMajorVersion20 = case OptMajorVersion of 20 -> "true"; _ -> "false" end.
+os:putenv("COUCHDB_OTP_MAJOR_VERSION_20", OtpMajorVersion20).
```

Comment:
If we only need to check this in Erlang code, we can rely on `OTP_RELEASE` directly:

https://erlang.org/doc/reference_manual/macros.html#predefined-macros

https://github.com/apache/couchdb/blob/00811b4a9c4df56defb7456431d44abb7c0baa79/src/fabric/src/fabric2_util.erl#L386-L387